x-pack/plugins/security_solution/scripts/endpoint/event_filters/index.ts

```diff
@@ -76,7 +81,30 @@ const createEventFilters: RunFn = async ({ flags, log }) => {
   await pMap(
     Array.from({ length: flags.count as unknown as number }),
     () => {
-      const body = eventGenerator.generate();
+      let options: Partial<ExceptionListItemSchema> = {};
+      if (flags.withWildcardPaths) {
```

Comment:
so if this option is `true`, all entry generated will have wildcard paths - is that the goal?
Or should then be a mix of entries created - some with wildcard paths others with "normal" entries?

Reply:
Yeah, the idea was to switch on the flag and get filters with wildcard paths. It's simpler and allows for generating filters with and without wildcads.

Having a mixed set of filters is also a good idea. We won't need a flag then.  I will go ahead and make that change.

Reply:
done d987ba3e4ab8d0356d081efd6b170883ae56638e